concept/thing/statistics.rs

```diff
@@ -52,6 +52,8 @@ pub struct Statistics {
     encoding_version: StatisticsEncodingVersion,
     pub sequence_number: SequenceNumber,
 
+    pub total_count: u64,
```

Comment:
Total count of all things is now also kept track of, so we can know when to flush the query cache by % overall change